Corona-Warn-App/src/main/res/values-bg/strings.xml

```diff
@@ -34,12 +34,6 @@
     <!-- NOTR -->
     <string name="preference_initial_result_received_time"><xliff:g id="preference">"preference_initial_result_received_time"</xliff:g></string>
     <!-- NOTR -->
-    <string name="preference_risk_level_score"><xliff:g id="preference">"preference_risk_level_score"</xliff:g></string>
```

Comment:
shouldn't be excluded to not cause any merge conflicts as the translation delivery will "override" everything. In the newest translation delivery after this is merged this shouldn't be included. But as we'll get our next translation delivery tomorrow I would say we leave it as it is.

Reply:
We may have to do resolve some merge conflicts on the translation delivery.

Reply:
This applies for all languages != de